crates/nargo/src/resolver.rs

```diff
@@ -127,7 +128,7 @@ impl<'a> Resolver<'a> {
                 return Err(DependencyResolutionError::RemoteDepWithLocalDep { dependency_path });
             }
             let mut new_res = Resolver::with_driver(self.driver);
-            new_res.resolve_manifest(crate_id, dep_meta.manifest)?;
+            new_res.resolve_manifest(crate_id, dep_meta.manifest, pkg_root)?;
```

Comment:
This is incorrect. We want this to be the package root of the dependency not the original package.